src/rtc/mosaic_geobursts.py

```diff
@@ -271,26 +469,35 @@ def compute_weighted_mosaic_array(list_rtc_images, list_nlooks,
 
 
 
-def compute_weighted_mosaic_raster(list_rtc_images, list_nlooks, geo_filename,
-                                   geogrid_in=None, verbose=True):
+def mosaic_single_output_file(list_rtc_images, list_nlooks, geo_filename,
+                              mosaic_mode, scratch_dir='', geogrid_in=None,
+                              temp_files_list=None, verbose=True):
     '''
     Mosaic the snapped S1 geobursts
-    paremeters:
+
+    Parameters:
     -----------
         list_rtc: list
```

Comment:
```suggestion
    Parameters:
        list_rtc_images: list
```

Reply:
We are trying to follow numpy style for docstrings: https://numpydoc.readthedocs.io/en/latest/format.html#parameters